src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php

```diff
@@ -57,10 +54,6 @@ public function __construct($listener, $name, Stopwatch $stopwatch, EventDispatc
         if (null !== $name) {
             $this->name = $name;
         }
-
-        if (null === self::$cloner) {
-            self::$cloner = class_exists(ClassStub::class) ? new VarCloner() : false;
```

Comment:
I'd keep a static prop for caching the class_exists check, eg:
`self::$hasClassStub = class_exists(ClassStub::class);`